subsys/input/Kconfig

```diff
@@ -95,4 +95,11 @@ config INPUT_KEYMAP
 	help
 	  Enable the input keymap driver.
 
+config INPUT_DOUBLE_TAP
```

Comment:
Can we extend this as a multi-tap with a user-defined number of taps? I was also going to do the same thing but with more tap events that I need in my IoT project.

Reply:
+1, this seems like a good idea

Reply:
maybe add it as a followup? up to you folks really

Reply:
Okay for me, double-tap is a good starting point.

Reply:
yeah it just seems to make more sense to me to to have the specific feature developed by someone who actually needs it :-)